src/tsickle.ts

```diff
@@ -630,6 +638,21 @@ class Annotator extends ClosureRewriter {
     return toArray(reexports.keys());
   }
 
+  private writeModuleSpecifier(moduleSpecifier: ts.Expression) {
+    let moduleId = (moduleSpecifier as any).text;
+    if (this.tsOpts && this.host) {
+      const resolved = ts.resolveModuleName(moduleId, this.file.fileName, this.tsOpts, this.host);
```

Comment:
Can you throw in a comment in this block about what you're doing (something about "rewrite index references to contain /index" or whatever)